### DISCUSSION TITLE
How to undo "remove all" of smplayer playlist ?

### DISCUSSION THREAD

**Discussion by andrew-ysk** [smplayer-dev/smplayer#549] [General]
I have accidentally "remove all" of my playlist that i just added while browsing youtube video. This is the 3rd time , because the "remove selected" and "remove all" are just too damn near, one accidental move of mouse while clicking will delete the whole  playlist. 
Is there way to undo the ""remove all" ? i have many info youtube video in my playlist that i want to go through, now they are all gone AGAIN.

Is there an UNDO on playlist ? 
Thanks

**Comment by smplayer-dev** [comment]
I'm afraid there's no undo.
